packages/cursorless-engine/src/disabledComponents/DisabledTalonSpokenForms.ts

```diff
@@ -0,0 +1,11 @@
+import type { TalonSpokenForms } from "../scopeProviders/TalonSpokenForms";
+
+export class DisabledTalonSpokenForms implements TalonSpokenForms {
+  getSpokenFormEntries() {
+    return Promise.resolve(null);
```

Comment:
if you just throw here, I believe it will show you a message and fall back to defaults anyway. Any reason not to just do that?

Reply:
Because I don't want an error message every time. This is not an error this is an intended behavior.

Reply:
I don't think it's unreasonable to get an error message if they try to use custom spoken forms when they're unsupported. This only shows when you pull up scope side bar right?

Reply:
But they're not trying to use customs spoken forms exactly. they're just trying to use the scope visualizer. For a user to get an error message that is an implementation decision by a developer feels wrong.

Reply:
The intent behind not supplying spoken forms is to use default. This is not an error to my mind.

Reply:
Yes but they should know that those spoken forms are not correct

Reply:
Keep in mind that the scope side bar doesn't exist in the engine; it's defined in VSCode. So if we had some ide that wants to support scope sidebar without showing spoken forms, we could make it so that it doesn't request custom spoken forms. But to my mind, at the layer we're operating in this PR, the components would be functioning as intended if they complain when you ask them for custom spoken forms, because that's their job

Reply:
if nothing else, it ends up being a smaller PR, so there's always that 😄